backend/src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -1053,3 +1053,23 @@ databaseChangeLog:
               UPDATE ${database.defaultSchemaName}.test_order
               SET date_tested_backdate = date_tested_backdate + interval '7' HOUR
               WHERE date_tested_backdate is not null;
+  - changeSet:
+      id: add-swab-type-to-device-type
+      author: tbest@cdc.gov
+      comment: add a property to store the swab type per device
+      changes:
+        - addColumn:
+            tableName: device_type
+            columns:
+              - column:
+                  name: swab_type
+                  type: *string
+                  remarks: The swab type used for this device
+        - sql:
+            comment: set the swab type for existing devices to anterior nares swab code (445297001)
+            sql: |
+              UPDATE ${database.defaultSchemaName}.device_type
+              SET swab_type = '445297001'
```

```suggestion
                  value: "445297001" # swab of internal nose
```

This can just be a `value: 445297001` and it'll do the update for you. We could also use `defaultValue` and probably do the null constraint at the same time, but that seems like the wrong way to go.

Also this is not actually anterior nares any more, it's [swab of the internal nose](http://snomed.info/id/445297001). 😄